openshift_resources/velero/backup.py

```diff
@@ -114,19 +115,17 @@ class BackupStatus(BaseModel):
     warnings: Optional[int]
 
 
-class Backup(BaseModel):
-    apiVersion: Optional[str]
-    kind :str = Field(__qualname__)
-    metadata: Optional[ObjectMeta]
+class Backup(BaseResource):
     spec: Optional[BackupSpec]
     status: Optional[BackupStatus]
 
 
-class BackupBuilder:
+class BackupBuilder(ResourceBuilder):
 
     # TODO: define better
-    def __init__(self, *args, **kwargs):
-        self.resource = Backup,
-        # TODO: add client
-        self.client: DynamicClient # type DynamicClient
+    def __init__(self, **kwargs):
+        super(BackupBuilder, self).__init__()
+        self.resource = Backup(**kwargs)
 
+    def discover_api_version(self, api_group=None):
```

Comment:
maybe set the API group as a constant in the Resource Class instead of overwriting the method?

Reply:
ack